components/phpunit_bridge.rst

```diff
@@ -163,13 +163,9 @@ Trigger Deprecation Notices
 
 Deprecation notices can be triggered by using::
 
-    @trigger_error('Your deprecation message', E_USER_DEPRECATED);
+    trigger_deprecation('vendor-name/package-name', '5.1', 'Your deprecation message');
 
-Without the `@-silencing operator`_, users would need to opt-out from deprecation
-notices. Silencing by default swaps this behavior and allows users to opt-in
-when they are ready to cope with them (by adding a custom error handler like the
-one provided by this bridge). When not silenced, deprecation notices will appear
-in the **Unsilenced** section of the deprecation report.
+Where 5.1 is the version from which the deprecation starts. Note that the deprecation message can use the :phpfunction:`printf` format. In this case, you can pass placeholders as extra arguments after the deprecation message.
```

Comment:
missing quotes around 5.1 for clarity

Reply:
This section was also changed during merge (a2846b9f04b371bc3f1f11f87075b2c686799861)